### DISCUSSION TITLE
Using environment variable in "uses" keyword

### DISCUSSION THREAD

**Discussion by bukowa** [community/community#25787] [Actions]
<p>I would like to use environment variable in “uses” keyword of the job, like that, for ex. $GITHUB_SHA:<br>
Can i do that?</p>
<pre><code class="lang-auto">      - name: Create kind cluster with custom vars
        uses: helm/kind-action@$GITHUB_SHA
        with:
          cluster_name: "custom-name"
          config: ./tests/config.yaml
          wait: 61s
          log_level: DEBUG
</code></pre>

**Comment by Simran-B** [comment, marked as answer]
<p>It’s not supported, see <a href="https://github.community/t/env-variables-in-uses/17466/2" class="inline-onebox">Env variables in uses - #2 by chrispat</a></p>

**Comment by bukowa** [comment]
<p>Thanks it makes sense but maybe it could be enabled at global level, like in, per repository settings.</p>

**Comment by jsoref** [comment]
<p>You can <code>actions/checkout</code> the repository to some place and then use <code>uses:</code> w/ a local reference.</p>
